src/components/web-components/rank-card/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { StyleSheet } from 'react-native';
+import { ITheme } from '../../../core/theme/itheme';
+
+export default (theme: ITheme) =>
+    StyleSheet.create({
+        container: {
+            backgroundColor: theme.colors.rankCard,
+            marginRight: '16.84%',
+            marginLeft: '16.84%',
+            marginTop: 37,
+            height: 67,
```

Comment:
Also this is not quite scalable, you should use `paddingVertical` instead of `height`.

Also it would be easier if we use multiple of 8 as measurement.